python-aexpect.spec

```diff
@@ -59,20 +91,48 @@ sftp, telnet, among others.
 %endif
 
 %build
-%{__python} setup.py build
+%py2_build
+
+%if %{with_python3}
+%py3_build
+%endif
 
 %install
-%{__python} setup.py install --root %{buildroot} --skip-build
+%py2_install
+# move and symlink python2 version-specific executables
+mv %{buildroot}%{_bindir}/aexpect-helper %{buildroot}%{_bindir}/aexpect-helper-%{python2_version}
+ln -s aexpect-helper-%{python2_version} %{buildroot}%{_bindir}/aexpect-helper-2
+
+%if %{with_python3}
+%py3_install
+mv %{buildroot}%{_bindir}/aexpect-helper %{buildroot}%{_bindir}/aexpect-helper-%{python3_version}
+ln -s aexpect-helper-%{python3_version} %{buildroot}%{_bindir}/aexpect-helper-3
+%endif
+
+# use python2 for unversioned executable
+ln -s aexpect-helper-%{python2_version} %{buildroot}%{_bindir}/aexpect-helper
```

Comment:
That makes `python3-aexpect` unusable without `python2-aexpect`, because `aexpect` only executes `aexpect-helper`. Would you please also add patch to first try the python-specific version first?